chef/cookbooks/crowbar/recipes/crowbar-db-dump.rb

```diff
@@ -106,6 +110,10 @@
       done
     EOF
   end
+  # Deletes postgres data dir in non-HA scenario
+  directory "/var/lib/pgsql/data" do
+    action :delete
```

Comment:
> I'm not convinced we would really hit any issue with disk space (do we have any figure that would indicate that?)

@vuntz I don't have any numbers of course. But we might (very roughly) need 3x the diskspace for the database after the upgrade than we did before. (SQL dump + old database + new database) I wouldn't bet that all systems that are upgraded have that much of a reserve. I guess it's another thing to document then.